app/src/main/java/org/simple/clinic/patient/recent/RecentPatientsView.kt

```diff
@@ -25,51 +33,21 @@ class RecentPatientsView(context: Context, attrs: AttributeSet) : RecyclerView(c
     layoutManager = LinearLayoutManager(context)
     adapter = groupAdapter
 
-    addDummyData()
-  }
-
-  private fun addDummyData() {
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish Acharya",
-        age = 43,
-        lastBp = "140/90",
-        gender = Gender.TRANSGENDER
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish24 Acharya",
-        age = 43,
-        lastBp = "141/90",
-        gender = Gender.MALE
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish3 Acharya",
-        age = 43,
-        lastBp = "142/90",
-        gender = Gender.FEMALE
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish4 Acharya",
-        age = 43,
-        lastBp = "145/90",
-        gender = Gender.TRANSGENDER
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish55 Acharya",
-        age = 43,
-        lastBp = "149/90",
-        gender = Gender.MALE
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish6 Acharya",
-        age = 43,
-        lastBp = "141/90",
-        gender = Gender.FEMALE
-    ))
-    groupAdapter.add(RecentPatientItem(
-        name = "Anish7 Acharya",
-        age = 43,
-        lastBp = "142/90",
-        gender = Gender.TRANSGENDER
-    ))
+    userSession.requireLoggedInUser()
```

Comment:
This should be done in a controller. Are you going to be doing this in the PR that completes this feature?

Reply:
Yup. This story is essentially to ensure that fetching recent patients is working as expected. The UI parts will be done in the associated UI story.